nautobot/core/settings.py

```diff
@@ -1092,3 +1092,22 @@ def silk_request_logging_intercept_logic(request):
 
 
 SILKY_INTERCEPT_FUNC = silk_request_logging_intercept_logic
+
+#
+# Kubernetes settings variables
+#
+
+# File path to the config file used to load and configure the kubernetes cluster
+KUBERNETES_CONFIG_FILE = os.getenv("NAUTOBOT_KUBERNETES_CONFIG_FILE", "./development/kind-kube-config")
```

Comment:
Wonder if this should just be `settings.KUBECONFIG` since the standard environment variable is just `$KUBECONFIG`.